### DISCUSSION TITLE
Custom Detections Only

### DISCUSSION THREAD

**Discussion by argwfm** [Security-Onion-Solutions/securityonion#13474] [2.4] [answered]
### Version

2.4.90

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

32

### Storage for /

256

### Storage for /nsm

512

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Is there an preferred way to disable all external (ET*) rules and use only custom? With the new bulk actions (TY) it's easier to disable the Emerging Threats rules, but hoping to save the periodical disabling of new rules as they are released. Would a regex to disable all ET* rules be the best option here, or is there a better way?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by InfosecGoon** [comment, marked as answer]
You could use a regex to disable all of the ET* rules -- this would be applied in the Administration --> Configuration interface. The option is under soc –> config –> server –> modules –> suricataengine –> disableRegex.

Documentation: https://docs.securityonion.net/en/2.4/nids.html#enabling-and-disabling-with-regex

**Reply by argwfm** [reply]
@InfosecGoon, thank you!
